sdk/src/main/java/com/hashicorp/nomad/javasdk/CSIPluginsApi.java

```diff
@@ -0,0 +1,87 @@
+package com.hashicorp.nomad.javasdk;
+
+import com.hashicorp.nomad.apimodel.CsiPlugin;
+import com.hashicorp.nomad.apimodel.CsiPluginListStub;
+import org.apache.http.client.utils.URIBuilder;
+
+import javax.annotation.Nullable;
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * API for querying for information about CSI plugins,
+ * exposing the <a href="https://www.nomadproject.io/api-docs/plugins/">plugins</a> functionality of the
+ * <a href="https://www.nomadproject.io/docs/http/index.html">Nomad HTTP API</a>.
+ */
+public class CSIPluginsApi extends ApiBase {
+
+    CSIPluginsApi(NomadApiClient apiClient) {
+        super(apiClient);
+    }
+
+    /**
+     * Lists CSI plugins.
+     *
+     * @param options         options controlling how the request is performed
+     * @throws IOException    if there is an HTTP or lower-level problem
+     * @throws NomadException if the response signals an error or cannot be deserialized
+     * @see <a href="https://www.nomadproject.io/api-docs/plugins/#list-plugins">{@code GET /v1/jobs}</a>
```

Comment:
This points to jobs: `{@code GET /v1/jobs}` (similar for the rest of CSI APIs)

Reply:
whoops, thanks. these are boilerplate, which means there's always a high probability of copy-paste errors on my part. thanks!